fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/generator/ArbitraryGeneratorContext.java

```diff
@@ -107,6 +111,30 @@ public <T extends Annotation> Optional<T> findAnnotation(Class<T> annotationClas
 		return this.getResolvedProperty().getAnnotation(annotationClass);
 	}
 
+	public <T extends Annotation> List<T> findAnnotations(Class<T> annotationClass) {
+		List<T> results = this.getResolvedProperty().getAnnotations().stream()
+			.filter(it -> annotationClass.isAssignableFrom(it.annotationType()))
+			.map(annotationClass::cast).collect(toList());
+
+		Repeatable repeatable = annotationClass.getAnnotation(Repeatable.class);
+		if (repeatable != null) {
+			Class<? extends Annotation> containerClass = repeatable.value();
+			this.getResolvedProperty().getAnnotations().stream()
+				.filter(it -> containerClass.isAssignableFrom(it.annotationType()))
+				.findFirst()
+				.ifPresent(container -> {
+					try {
+						Method valueMethod = container.annotationType().getDeclaredMethod("value");
+						T[] values = (T[])valueMethod.invoke(container);
+						results.addAll(Arrays.asList(values));
+					} catch (Exception ignored) {
+					}
+				});
+		}
+
+		return results;
+	}
```

Comment:
It is not for common case.

You can not confirm that user uses the annotation `@Repeatable` , the name of property `value` .

I think it should be removed.

Reply:
@seongahjo 
Ah, I understand! Instead of defining a new `findAnnotations` in `ArbitraryGeneratorContext`, should I handle duplicate annotations within `ArbitraryGeneratorContext`?

Reply:
Does `this.getResolvedProperty().getAnnotations()` not return the duplicate annotations?

Could you make a test to show when the duplicate annotation is an issue?

Reply:
```java
@Getter
@Setter
public class TestSpec {
	@Max(value = 50)        // interface jakarta.validation.constraints.Max
	private BigDecimal a;

	@Max(value = 100)       // multiple @Max annotations - internally processed as container annotation
	@Max(value = 50)
	private BigDecimal b;     

	@Max.List({             // interface jakarta.validation.constraints.Max$List
		@Max(value = 100),
		@Max(value = 50)
	})
	private BigDecimal c;
}
```
Duplicate annotations appear to be automatically processed in the same way as the annotations in field c.

Therefore, I implemented `findAnnotations` to process the contents within `Max$List` in the same way as regular duplicate annotations, as shown above.

Without the `Repeatable` processing in `findAnnotations`, if we only use `this.getResolvedProperty().getAnnotations()`, we would need to search twice using `getAnnotations(Max.class)` and `getAnnotations(Max.List.class)`. This is because we need to consider both cases where there is a single annotation and multiple annotations.
Since `getAnnotationsByType` cannot be used in the current implementation, I chose to implement `findAnnotations` to handle both single annotations and container annotations in a unified way.

Reply:
Does the JSR-303 support multiple annotations on the same property within the same group?

I only found the applying multiple constraint within the different group.
https://beanvalidation.org/1.0/spec/#constraintsdefinitionimplementation-multipleconstraints

Reply:
Since the @Repeatable annotation allows multiple annotations to be applied, it was confirmed during testing that all validation annotations (including duplicates) are processed as expected.

However, JSR-303 does not define a specific approach for handling duplicate annotations, meaning implementations can vary.

Given this, rather than implementing complex logic to handle duplicates, it may be more practical to revert to the original approach of simply checking for the presence of annotations.